controller/user.go

```diff
@@ -0,0 +1,60 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-cluster/app"
+	"github.com/fabric8-services/fabric8-common/token"
+	"github.com/goadesign/goa"
+
+	"github.com/fabric8-services/fabric8-cluster/application"
+	"github.com/fabric8-services/fabric8-common/errors"
+	"github.com/fabric8-services/fabric8-common/log"
+)
+
+// UserController implements the user resource.
+type UserController struct {
+	*goa.Controller
+	app application.Application
+}
+
+// NewUserController creates a user controller.
+func NewUserController(service *goa.Service, app application.Application) *UserController {
+	return &UserController{Controller: service.NewController("UserController"), app: app}
+}
+
+// Clusters runs the clusters action.
+func (c *UserController) Clusters(ctx *app.ClustersUserContext) error {
+	tokenMgr, err := token.ReadManagerFromContext(ctx)
+	if err != nil {
+		return app.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
+	}
+	identityID, err := tokenMgr.Locate(ctx)
+	if err != nil {
+		return app.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
+	}
```

Comment:
I've created a PR to simplify locating the identity ID in the context - https://github.com/fabric8-services/fabric8-common/pull/70
When it's merged you can replace lines 26-33 by:
```
identityID, err := auth.LocateIdentity(ctx)
if err !=nil {
    // Log the error
   ...
    return app.JSONErrorResponse(ctx, err) // No need to return 401 here. If we failed to load the TM then it's 500.
}
```

Reply:
@alexeykazakov I believe that we should return a `401` error if the request is missing a token or if the token is not valid (missing claim, invalid signature, etc.). Can we discuss about this?

Reply:
@xcoulon yes we should. But it will be handled by the JWT middleware. We don't have to duplicate it in the controller.
If the token is good (JWT middleware is happy with that) but we failed to get the token manager then it's 500 for us. It means we have a bug in our code. It's not a problem with the request.